### PR TITLE
Integrate codegen into application

### PR DESCRIPTION
**Pull request by wscalf**
- Added gRPC service
    - Converted HTTP endpoint to wrapper for gRPC service
    - Added gRPC endpoint
    - Updated proto files to be API-compatible with current spec
    - Confirmed curl commands work with new build

### CODE REVIEW

File: host/GRpc.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	"sync"
+
+	"github.com/golang/glog"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/metadata"
+)
+
+// GRpc represents a GRpc host service
+type GRpc struct {
```
Comment (DGuhr) [inline host/GRpc.go:21]: is the uppercase GR here and in the file etc. intended? if so, why? also we should think about naming a bit i think. would love to do a refactoring collab, just ping me

Comment (Rajagopalan-Ranganathan) [reply host/GRpc.go:21]: upper case: usually means it is "exposed" . I am not sure if that is intended here. but filenames in go, the best practice is to use lower case to start (refer packagenames section in: https://go.dev/doc/effective_go#package-names)

Comment (DGuhr) [reply host/GRpc.go:21]: then it should be `Grpc` not `GRpc` right? (and yes, I know that ;) )

Comment (wscalf) [reply host/GRpc.go:21]: It was Grpc originally. The linter said to change it. >.>

**Review by DGuhr — APPROVED** [review]

**Pull request merged**